Automattic/wp-calypso
# Signup: Update colors and background

#### Changes proposed in this Pull Request

* Adds a dark blue background color, with white or light gray text
* Adds two fixed shapes to the background
* Both these changes bring us closer to parity with the new WP.com home page and marketing materials

Before:

<img width="1440" alt="screen shot 2018-10-12 at 1 41 29 pm" src="https://user-images.githubusercontent.com/2124984/46884986-95abaa00-ce24-11e8-8ff8-5c57b080a6b7.png">

After:

<img width="1440" alt="screen shot 2018-10-12 at 1 41 21 pm" src="https://user-images.githubusercontent.com/2124984/46884985-95abaa00-ce24-11e8-9f22-ee3cddec1c33.png">

#### Testing instructions

* Switch to this PR and navigate to `/start/`
* Check to see if the colors have changed throughout signup
* Check any special flows (ie. `/start/rebrand-cities/`, or select "sell products" at the start)
* Check elsewhere on WP.com to make sure the colors are not affected (things like `/add/domains/` or `/plans/`, for example)
